### PR TITLE
Glance over cinder (iSCSI) backend

### PR DESCRIPTION
**Pull request by dsariel**
According to
  https://github.com/openstack-k8s-operators/glance-operator/blob/e296d336e2d0ff0c7f0a5a41a1ba839532864e5e/config/samples/backends/cinder/glance-common/glance.yaml

[OSPRH-7710](https://issues.redhat.com//browse/OSPRH-7710)

### CODE REVIEW

**Comment by dsariel** [comment]
it was verified here: https://url.corp.redhat.com/glance-over-cinder

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 1]
```diff
@@ -38,22 +38,32 @@ data:
...
+      cinder_store_user_name = {{ .ServiceUser }}
+      cinder_store_password = {{ .ServicePassword }}
+      cinder_store_project_name = service
+      cinder_catalog_info = volumev3::internalURL
+      # TODO: Enable multipath when OSPRH-7393 is resolved
+      # cinder_use_multipath = true
+      [oslo_concurrency]
+      lock_path = /var/lib/glance/tmp
+      [database]
+      mysql_wsrep_sync_wait = 1
```
Comment (tosky) [inline examples/dt/uni07eta/control-plane/service-values.yaml:58]: Maybe this [database] block is not needed anymore after https://github.com/openstack-k8s-operators/architecture/pull/297

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 2]
```diff
@@ -38,22 +38,32 @@ data:
   glance:
     customServiceConfig: |
       [DEFAULT]
-      debug=True
-      enabled_backends = default_backend:swift
-
+      enabled_backends = default_backend:cinder
+      debug = true
```
Comment (tosky) [inline examples/dt/uni07eta/control-plane/service-values.yaml:42]: I would keep the order, with debug = True as first argument

**Comment by fultonj** [comment]
setting needs-info given Luigi's questions

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 1 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane/service-values.yaml:58]: done

**Review by abays — APPROVED** [review]
/lgtm
/approve

**Comment by dsariel** [comment]
re-verified after 

results are here: https://url.corp.redhat.com/142#note_11297910

Merging. Thanks a lot @abays @fultonj @tosky

**Pull request merged**